index.template.html

```diff
@@ -46,6 +46,7 @@
 		<!-- meta name="mobile-web-app-capable" content="yes" / -->
 		<!-- meta name="apple-mobile-web-app-capable" content="yes" / -->
 		<meta name="robots" content="noindex" />
+		<meta name="google-signin-client_id" content="912270888098-jjnre816lsuhc5clj3vbcn4o2q7p4qvk.apps.googleusercontent.com">
```

Comment:
Hardcoding this here is kind of lazy, but rewriting the update script to pull from `config/config.inc.php` sounds annoying. :<